scripts/start.sh

```diff
@@ -5,9 +5,24 @@ set -e
 CUR_DIR=$(cd "$(dirname "$0")" && pwd)
 BASE_DIR=$(dirname "$CUR_DIR")
 
+min() {
+    echo $(( $1 < $2 ? $1 : $2 ))
+}
+
+workers=$(min 30 ${NB_WORKERS_ANTARES:-17}) # default to 17 and max is 30
+
 if [ -z "$1" ] ; then
   sh $CUR_DIR/pre-start.sh
-  gunicorn --config $BASE_DIR/conf/gunicorn.py --worker-class=uvicorn.workers.UvicornWorker antarest.wsgi:app
+  pids=() # Initialize empty array to store background process IDs
+  for ((i=0; i<workers; i++))
+  do
+    uvicorn antarest.wsgi:app --host 0.0.0.0 --port $((5000 + $i)) --log-level info --timeout-keep-alive 600 &
```

Comment:
Again not really familiar with the subject but why do we have different ports ? Also shouldn't we lower the tiemout-keep-alive ?

Reply:
we want each `uvicorn` to run in a different process with its own port to be considered as a separate worker.

Reply:
keep alive was one day with gunicorn, seems very high compared to standard values but we could keep it for now.

Using different ports is the goal of this PR, it will allow an external component to do the load balancing for us (nginx in our case).